datalad/customremotes/archive.py

```diff
@@ -256,7 +255,6 @@ def req_CHECKPRESENT(self, key):
         TODO: just proxy the call to annex for underlying tarball
 
         Replies
-        -------
```

Comment:
Here is the list of supported headings:

http://sphinx-doc.org/latest/ext/napoleon.html#docstring-sections

I wasn't sure what "repies" refers to.